Sources/MapboxCoreNavigation/Locale.swift

```diff
@@ -23,20 +23,13 @@ extension Locale {
      Returns a `Locale` from `preferredLocalLanguageCountryCode`.
      */
     public static var nationalizedCurrent = Locale(identifier: preferredLocalLanguageCountryCode)
-    
+
     public static var usesMetric: Bool {
```

Comment:
Might as well deprecate this method if it’s as simple as writing `Locale.current.usesMetric`.